### PR TITLE
New request strategy

### PR DESCRIPTION
**Pull request by HarmenKroon**
As a virtual human, I want to react intelligently to requests received, so that I make the best decisions for the stakeholder.

This PR also adds a new version of the environment

### CODE REVIEW

File: src/main/goal/nl/tudelft/ti2806/request.mod2g [hunk 1]
```diff
@@ -1,61 +1,124 @@
...
-		)
-	then popup_answer(RequestID, AnswerID).
+			RequestType == 'BUY_LAND')
+			then {
+				
+			% description:
+			% Decide if we want to sell a piece of land
+			if
+				bel(
+					marktprijs(Huidigemarktprijs),
```
Comment (nskartoredjo) [inline src/main/goal/nl/tudelft/ti2806/request.mod2g]: change to English

**Comment by jannelie** [comment]
Approved

**Comment by HarmenKroon** [comment]
Sell_Land now also takes needed land into consideration

**Comment by nskartoredjo** [comment]
Approved

**Comment by ardwennem2** [comment]
Approved

**Pull request merged**